### ISSUE TITLE
Remove the z/OS pre-req of npm from the docs as it's not required for runtime

### ISSUE THREAD

**Issue by Joe-Winchester** [zowe/docs-site#446]
## Description
The pre-req in the installing on z/OS calls out that people need npm 5.4 or higher.  This isn't correct as only node is required to run zowe on z/OS

## Pages to Update
https://zowe.github.io/docs-site/latest/user-guide/systemrequirements.html#overview

## Screenshots
![image](https://user-images.githubusercontent.com/28100302/55196037-11689b80-51a6-11e9-8773-61bd4643e6fa.png)

![image](https://user-images.githubusercontent.com/28100302/55196086-44129400-51a6-11e9-9fc4-0734139e4a87.png)

https://openmainframeproject.slack.com/archives/CC08782AG/p1553768549000500

## Expected behavior
Docs just call out the need for node runtime and don't call out the need for node (which is a package installer)

## Additional context
When/why did we call out the need for npm ?  I recall that we used to only call out node as pre-req so something changed.  I wanted to be sure that I'm not missing something

**Comment by Joe-Winchester** [comment]
@nannanli - have the docs always included the section asking for npm 5.4 to be a pre-req ?  We're trying to validate whether it's needed (I don't recall this section in early docs) so it must have come from somewhere and I want to make sure it's not needed before we update the docs.  Tks

**Comment by nannanli** [comment]
Hi Joe @Joe-Winchester , I checked the 0.9.0 doc and some older versions, previously it was documented as follows: 
<img width="779" alt="sysreq-0 9 1" src="https://user-images.githubusercontent.com/14340789/55222123-12e9a080-5246-11e9-84a5-aad1cd736e7f.png">

```
npm 5.4 or later **for building Zowe Application Framework applications**.
To update npm, issue the following command:
npm install -g npm
```
The highlighted words got lost when a consolidated list of sysreq was pulled together back in 1.0.0. There is not an issue requesting to move the highlighted words so it should be a mistake.

Reading the problem, shall we remove the need of npm or add back that it's required for building applications? This is the link to the sysreq topic on doc-staging: https://github.com/zowe/docs-site/blob/docs-staging/docs/user-guide/systemrequirements.md#system-requirements, but the change will need to be to backported to earlier releases.

**Comment by Joe-Winchester** [comment]
Yes please remove all references to npm.  @petr-galik are you OK with that ?

**Comment by nannanli** [comment]
This issue is fixed via #448. Closing.
